standards/core/lightpush.md

```diff
@@ -0,0 +1,88 @@
+---
+title: WAKU2-LIGHTPUSH
+name: Waku v2 Light Push
+status: draft
+editor: Zoltan Nagy <zoltan@status.im> 
+contributors: 
+  - Hanno Cornelius <hanno@status.im>
+  - Daniel Kaiser <danielkaiser@status.im>
+  - Oskar Thorén <oskarth@titanproxy.com>
+---
+
+previous version: `/vac/waku/lightpush/2.0.0-beta1` [19/WAKU2-LIGHTPUSH](https://rfc.vac.dev/waku/standards/core/19/lightpush)
+
+---
+**Protocol identifier**: `/vac/waku/lightpush/3.0.0`
+
+## Motivation and Goals
+
+Light nodes with short connection windows and limited bandwidth wish to push messages to other nodes in the Waku network to request message services.
+A common use case is to request that service node to publish the message to a `11/WAKU2-RELAY` pubsub topic.
+Additionally, there is sometimes a need for confirmation that a message has been received "by the network"
+(here, at least one node).
+
+`WAKU2-LIGHTPUSH` is a request/response protocol for this.
+
+## Payloads
+
+```protobuf
+syntax = "proto3";
+
+message LightPushRequest {
+        string request_id = 1;
+    uin32 request_type = 10;  // reserved for future request type selection, currently it is always RELAY (0)
+    string topic = 20;
+    WakuMessage message = 21;
+}
+
+message LightPushResponse {
+    string request_id = 1;
+    uint32 status_code = 10; // non zero in case of failure, see appendix
+    optional string status_desc = 11;
+    uint32 relay_peer_count = 12;
+}
+```
+
+### Message Relaying
+
+Nodes that respond to `LightPushRequest` MUST either relay the encapsulated message via [11/WAKU2-RELAY](https://rfc.vac.dev/waku/standards/core/11/relay) protocol on the specified `pubsub_topic` or `shard` ([WAKU2-RELAY-SHARDING](https://github.com/waku-org/specs/blob/master/standards/core/relay-sharding.md)) depending on the network configuration.
```

Comment:
I would avoid using `shard` on this layer. The use of "either" here doesn't make sense as we don't provide an alternative.

Reply:
I reformatted this section.